freeipa/src/main/java/com/sequenceiq/freeipa/flow/stack/modify/proxy/action/ModifyProxyConfigFailedAction.java

```diff
@@ -0,0 +1,48 @@
+package com.sequenceiq.freeipa.flow.stack.modify.proxy.action;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.common.DetailedStackStatus;
+import com.sequenceiq.freeipa.flow.stack.StackEvent;
+import com.sequenceiq.freeipa.flow.stack.StackFailureEvent;
+import com.sequenceiq.freeipa.flow.stack.modify.proxy.ModifyProxyConfigContext;
+import com.sequenceiq.freeipa.flow.stack.modify.proxy.selector.ModifyProxyConfigStateSelector;
+import com.sequenceiq.freeipa.service.operation.OperationService;
+import com.sequenceiq.freeipa.service.stack.StackUpdater;
+
+@Component("ModifyProxyConfigFailedAction")
```

Comment:
This state-action relation is defined in the state enum's action, so it works this way too and I think it is more readable

Reply:
ok

Reply:
The automatic bean name would be modifyProxyConfigFailedAction (not the starting lowercase), so it is needed because the flow framework would not find that bean